src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -1042,17 +1057,34 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
 
                                             $verticalBar = '|'
                                             $posmsg += ""${accentColor}${headerWhitespace}Line ${verticalBar}${newline}""
-                                            $line = $myinv.Line
-                                            $highlightLine = $myinv.PositionMessage.Split('+').Count - 1
-                                            $offsetLength = $myinv.PositionMessage.split('+')[$highlightLine].Trim().Length
+
+                                            if ($useTargetObject) {
+                                                $line = $_.TargetObject.LineText.Trim()
+                                                $indexOfShould = $line.IndexOf('should',[System.StringComparison]::OrdinalIgnoreCase)
```

Comment:
this looks more and more like a specific behavior for Pester

Reply:
If we don't want to optimize for Pester, I can just have this point to the beginning of the line instead at should since there isn't any information for the offset